packages/chaire-lib-backend/src/models/db/tokens.db.queries.ts

```diff
@@ -109,11 +126,27 @@ const getUserByToken = async (token: string) => {
     }
 };
 
+async function cleanExpiredApiTokens() {
+    try {
+        const tokenLifespan = new Date();
+        const rowsToDelete = await knex(tableName).where('expiry_date', '<', tokenLifespan);
+        for (const row of rowsToDelete) {
```

Comment:
We cannot do a delete directly?

Something in SQL like DELETE FROM ... WHERE expiry_data < NOW() ;

Reply:
Yes. By proceeding like this, we remove the use for id verification.